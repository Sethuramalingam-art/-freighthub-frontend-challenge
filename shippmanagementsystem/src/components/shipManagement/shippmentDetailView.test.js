import React from 'react';
import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import ShippmentDetailView from './shippmentDetailView';

const mockStore = configureStore([]);
describe("Connected ShippmentDetailView Component", () => {
    let store;
  let component;
  beforeEach(() => {
    store = mockStore({
    });
    component = renderer.create(
      <Provider store={store}>
        <ShippmentDetailView />
      </Provider>
    );
  });
  it("should render ShippmentDetailView component", () => {
    expect(component).toMatchSnapshot();
  });
});