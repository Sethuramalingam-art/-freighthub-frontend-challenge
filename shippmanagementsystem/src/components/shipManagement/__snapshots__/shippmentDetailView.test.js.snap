// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connected ShippmentDetailView Component should render ShippmentDetailView component 1`] = `undefined`;
